jetbrains-rider/src/software/aws/toolkits/jetbrains/services/lambda/dotnet/DotNetSamDebugSupport.kt

```diff
@@ -280,17 +277,12 @@ class DotNetSamDebugSupport : SamDebugSupport {
         sessionModel: DotNetDebuggerSessionModel,
         outputEventsListener: IDebuggerOutputListener
     ): XDebugProcessStarter {
-        val consoleKind = ConsoleKind.ExternalConsole
-        (executionConsole as? ConsoleView)
-            ?.print(
-                "Input/Output redirection disabled: ${consoleKind.message}${System.lineSeparator()}",
-                ConsoleViewContentType.SYSTEM_OUTPUT
-            )
 
         val fireInitializedManually = env.getUserData(DotNetDebugRunner.FIRE_INITIALIZED_MANUALLY) ?: false
 
         return object : XDebugProcessStarter() {
             override fun start(session: XDebugSession): XDebugProcess =
+                // TODO: Update to use 'sessionId' parameter in ctr when min SDK version is 193.
```

Comment:
Our pattern is to mark these at the end of the comment with a standard string, in this case it would be `Update to use 'sessionId' parameter in constructor FIX_WHEN_MIN_IS_193`